### PR TITLE
Create tommy-shelby.json

### PR DESCRIPTION
**Pull request by prbhanu**

### CODE REVIEW

**Review by TheBotlyNoob — DISMISSED** [review]
:+1:

File: domains/tommy-shelby.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Tommy Shelby portfolio",
+  "repo": "https://github.com/prbhanu/tommy-shelby",
+  "owner": {
+    "username": "prbhanu",
+    "email": "prreddy494@gmail.com",
+    "twitter": "@prreddy05413731"
+  },
+  "record": {
+    "CNAME": "https://prbhanu.github.io/tommy-shelby/"
```
Comment (TheBotlyNoob) [inline domains/tommy-shelby.json:10]: the cname cannot have a `/`. if you need to have a `/`, use a `URL` record instead.

**Review by TheBotlyNoob — CHANGES_REQUESTED** [review]

**Comment by mtgsquad** [comment]
Oh.

**Comment by prbhanu** [comment]
> Oh.

whats wrong?

File: domains/tommy-shelby.json (hunk 1 above)
Comment (prbhanu) [reply domains/tommy-shelby.json]: Can u check now?

File: domains/tommy-shelby.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Tommy Shelby portfolio",
+  "repo": "https://replit.com/@prreddy1/tommyshelbydev-#index.html",
+  "owner": {
+    "username": "prreddy1",
```
Comment (MaskDuck) [inline domains/tommy-shelby.json:5]: insert your github username, not replit

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Pull request closed without merging**